### PR TITLE
ECIP-[TBD/1105]: Restricted Chain Context Type Transactions (Draft)

### PR DESCRIPTION
**Pull request by meowsbits**
This is submitting a `Draft` status `Standards > Core` proposal for consideration.

#### Simple Summary

Add a transaction type which contains a _Chain Context_ value restricting a transaction's validity
to a chain meeting certain contextual requirements.

This proposal is also drafted at https://github.com/ethereum/EIPs/blob/master/EIPS/eip-3534.md.

### CODE REVIEW

**Comment by bobsummerwill** [comment]
Please could you provide more context or examples?

It is unclear to me what specific use-cases for this functionality might look like end-to-end.

File: _specs/eip-restricted_chain_context_txs.md [hunk 1]
```diff
@@ -0,0 +1,180 @@
+---
+ecip: TBD
+title: Restricted Chain Context Type Transactions
+author: Isaac Ardis <isaac.ardis@gmail.com>
+discussions-to: TBD
```
Comment (meowsbits) [inline _specs/eip-restricted_chain_context_txs.md:5]:
```suggestion
discussions-to: https://github.com/ethereumclassic/ECIPs/issues/422
```

**Comment by meowsbits** [comment]
@bobsummerwill I've just added a [Discussions-To](https://github.com/ethereumclassic/ECIPs/issues/422) link. Please let's move the conversation there.

File: _specs/ecip-restricted_chain_context_txs.md [hunk 2]
```diff
@@ -0,0 +1,318 @@
+---
+ecip: TBD
```
Comment (meowsbits) [inline _specs/ecip-restricted_chain_context_txs.md:2]:
```suggestion
ecip: 1105
```

File: _specs/ecip-restricted_chain_context_txs.md [hunk 3]
```diff
@@ -0,0 +1,313 @@
...
```
Comment (q9f) [inline _specs/ecip-restricted_chain_context_txs.md:2]:
```suggestion
ecip: 1106
```

File: _specs/ecip-restricted_chain_context_txs.md [hunk 4]
```diff
@@ -0,0 +1,313 @@
+---
+ecip: 1105
```
Comment (meowsbits) [inline _specs/ecip-restricted_chain_context_txs.md:2]:
```suggestion
ecip: 1106
```

**Review by q9f — APPROVED** [review]

**Review by Conanza00 — APPROVED** [review]

**Review by Conanza00 — APPROVED** [review]

**Review by Conanza00 — APPROVED** [review]

**Pull request merged**